starter-core/src/main/java/io/micronaut/starter/feature/graalvm/GraalVMFeatureValidator.java

```diff
@@ -35,7 +36,7 @@ public void validatePreProcessing(Options options, ApplicationType applicationTy
 
     @Override
     public void validatePostProcessing(Options options, ApplicationType applicationType, Set<Feature> features) {
-        if (features.stream().anyMatch(f -> f instanceof GraalVM)) {
+        if (features.stream().anyMatch(f -> f instanceof GraalVM || f instanceof GraalVMDockerRegistryWorkflow)) {
```

Comment:
I do not rely on graalvm feature now in any of the workflow features, but for graal the validator is reused.